.github/workflows/main.yml

```
@@ -131,3 +131,10 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
           NPM_TOKEN: ${{ secrets. NPM_TOKEN }}
+
+      - name: 🌐Publish dotLottie-viewer
+        if: success()
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
```

We don't have CUSTOM_GITHUB_TOKEN in this repo
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```